### DISCUSSION TITLE
Quickdraw Badge

### DISCUSSION THREAD

**Discussion by danilrannev** [community/community#39818] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi

How can I get the quickdraw badge? Please help me.

Regards

**Comment by slava-shkarlet** [comment, marked as answer]
Closed an issue or a pull request within 5 min of opening
Read this: https://github.com/Schweinepriester/github-profile-achievements

**Comment by futuresea-dev** [comment]
Closed an issue or a pull request within 5 min of opening

https://savelev.medium.com/improving-github-profile-with-highlights-and-achievements-be5fcdd558d
